src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -25,18 +26,20 @@
 
 (defn- render-property [library-schema library-attributes property-key]
   [:div {:style {:display "flex" :padding "0.5em 0" :borderBottom (str "2px solid " (:line-default style/colors))}}
-   [:div {:style {:flexBasis "33%" :fontWeight "bold" :paddingRight "2em"}}
+   [:div {:style {:flexBasis "40%" :fontWeight "bold" :paddingRight "2em"}}
     (get-in library-schema [:properties property-key :title])]
-   [:div {:style {:flexBasis "67%"}}
-    (render-value (get library-attributes property-key))]])
+   [:div {:style {:flexBasis "60%"}}
+    (render-value (get library-attributes property-key))
+    (if (= property-key :library:diseaseOntology) [:span {:style {:fontStyle "italic"}} (str " (DOID: " (style/url-to-doid (get library-attributes :library:diseaseOntologyDOID)) ")")])]])
```

Comment:
this shouldn't look specifically for a property named "diseaseOntology" - it should be looking for any property enabled for ontology typeahead. Similarly, once we use another ontology, the "DOID_" prefix will be invalid.